gulpfile.js

```diff
@@ -137,8 +141,20 @@ gulp.task("watch", function () {
   gulp.watch(dirs.img + "/**/*.*", ["images"]);
 });
 
+gulp.task("make-war", function () {
+  var warFileName = packageInfo.name + "." + packageInfo.version + ".war";
+  return gulp.src("./dist/**/*")
+    .pipe(war({
+      welcome: "index.html",
+      displayName: "Marathon UI"
+    }))
+    .pipe(zip(warFileName))
+    .pipe(gulp.dest("."));
+});
+
 gulp.task("serve", ["default", "connect:server", "watch"]);
 gulp.task("livereload", ["default", "browsersync", "watch"]);
+gulp.task("war", ["default", "make-war"]);
```

Comment:
Eventually, yes - for the moment the main marathon build depends on its existence so I was going to do this in a separate PR.